web/src/components/ShareIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Capacitor } from '@capacitor/core';
+import { Share, Share2 } from 'lucide-react';
+import { DEFAULT_ICON_SIZE } from 'utils/constants';
+
+export function ShareIcon({
+  showIosIcon = defaultShouldShowIosIcon(),
+  iconSize = DEFAULT_ICON_SIZE,
+}: {
+  showIosIcon?: boolean;
+  iconSize?: number;
+}) {
+  return showIosIcon ? (
+    <Share data-test-id="iosShareIcon" size={iconSize} />
+  ) : (
+    <Share2 data-test-id="defaultShareIcon" size={iconSize} />
+  );
+}
```

Comment:
Should this be memoized? I don't think thee will ever change since one is platform dependant and the other is just the icon size.

Reply:
Yah, that's a great idea 👍